### DISCUSSION TITLE
自然生活的数学原理 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by yufree** [yufree/yufree.cn#304] [General]
自然生活的数学原理 - Miao Yu | 于淼 
 https://yufree.cn/cn/2018/02/13/naturalis-principia-mathematica-vitae/

**Comment by yufree** [comment]
`Originally posted by Tianxia Zhou on 2018-04-10 04:51:09
        `
这个写法很有趣。

**Comment by timoseven** [comment]
这个网站怎么一眼就是ai生成的，是不是用的trae？

**Reply by yufree** [reply]
第一版glm生成的，但数据库质量太差，老是偷懒用重复的推荐语，这版是opus重写的数据库，为了节省开支我要求agent调用haiku做脏活，但opus评价了下发现时效性对齐任务质量还是不行，也是偷懒，最后烧了我三分之一的周限额补上了，都是claudcode做的
